src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -232,14 +232,20 @@ def _create_role_assignment(cli_ctx, role, assignee, resource_group_name=None, s
                                          condition=condition, condition_version=condition_version)
 
 
-def list_role_assignments(cmd, assignee=None, role=None, resource_group_name=None,
+def list_role_assignments(cmd,
+                          assignee=None, assignee_object_id=None,
+                          role=None, resource_group_name=None,
                           scope=None, include_inherited=False,
                           show_all=False, include_groups=False, include_classic_administrators=False,
                           fill_principal_name=True):
-    '''
-    :param include_groups: include extra assignments to the groups of which the user is a
-    member(transitively).
-    '''
```

Comment:
`include_groups` should be declared in `_params.py`, not here. https://github.com/Azure/azure-cli/pull/602#discussion_r2020738332